etc/postgresql/values.yaml

```diff
@@ -32,7 +32,7 @@ image:
   ## It turns BASH and NAMI debugging in minideb
   ## ref:  https://github.com/bitnami/minideb-extras/#turn-on-bash-debugging
   ##
-  debug: true
+  debug: false
```

Comment:
Isn't it better to keep this true?

Reply:
For most production installations (and by default), I'd say no.